common/plugins/eu.esdihumboldt.hale.common.align/src/eu/esdihumboldt/hale/common/align/model/AlignmentUtil.java

```diff
@@ -1045,13 +1046,23 @@ public static void addValues(Group group, List<ChildContext> path,
 
 				// check all values
 				for (Object value : values) {
-					if (value instanceof Group) {
+					if (value instanceof InstanceDecorator) {
+						if (((InstanceDecorator) value).getValue() != null) {
+							collectedValues.add(((InstanceDecorator) value).getValue());
```

Comment:
Looking at the method signature it seems to me that the `onlyValues` parameter is what is used here to control if it is OK to return an `Instance` or rather it's value.
See here: https://github.com/halestudio/hale/blob/5cec368a3da576f45c6b100adc378611d12b54cc/common/plugins/eu.esdihumboldt.hale.common.align/src/eu/esdihumboldt/hale/common/align/model/AlignmentUtil.java#L1002

Do you have a hint from where `addValues` is called in your case and how it is determined what value is set for the `onlyValues` parameter?